bigtable-hbase-1.x-parent/bigtable-hbase-1.x-hadoop/pom.xml

```diff
@@ -71,6 +71,12 @@ limitations under the License.
       <version>${hbase1.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.cloud.bigtable</groupId>
+      <artifactId>bigtable-metrics-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
```

Comment:
I think this is belongs in the manually promoted section below : we aren't shading it, and the reason its here is so that it ends up in the deps